### PR TITLE
build(docs-app): update dependencies to latest Angular 15 version

### PR DESCRIPTION
**Pull request by cedricduffournet**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by cedricduffournet** [comment]
close/open to restart CI

**Review by timdeschryver — CHANGES_REQUESTED** [review]
Some changes (colors) we need to look at 

image

image

image

File: projects/ngrx.io/src/styles/2-modules/_code.scss [hunk 1]
```diff
@@ -1,13 +1,14 @@
+@use '@angular/material' as mat;
```
[the remark this answers is not in the document]
Comment (cedricduffournet) [reply projects/ngrx.io/src/styles/2-modules/_code.scss:1]: No I did it manually

**Comment by cedricduffournet** [comment]
thanks @timdeschryver I'll check that

**Comment by cedricduffournet** [comment]
@timdeschryver I restore textarea height and fix hamburger button pres. 
It seems that automatic labelling is not working today

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**